code/modules/mob/living/simple_animal/hostile/handofgod.dm

```diff
@@ -0,0 +1,119 @@
+/mob/living/simple_animal/hostile/handofgod
+	name = "Hand of God"
+	desc = "A giant hand.  Smells like a great meal."
+	icon = 'icons/mob/handofgod.dmi'
+	icon_state = "handofgod"
+	icon_living = "handofgod"
+	icon_dead = "handofgod"
+	turns_per_move = 5
+	response_help_continuous = "rubs"
+	response_help_simple = "rub"
+	response_disarm_continuous = "nudges"
+	response_disarm_simple = "nudge"
+	speed = 0
+	maxHealth = 300
+	health = 300
+
+	harm_intent_damage = 0  //Mere mortal hands can't damage the visage of a god
+	obj_damage = 500  //He isn't fucking around
+	melee_damage_lower = 1
+	melee_damage_upper = 20  //He rolls the dice
+	attack_verb_continuous = "subjugates"
+	attack_verb_simple = "subjugate"
+	attack_sound = 'sound/magic/wandodeath.ogg'
+	speak_emote = list("deliciously declares")
+
+	//Hamburger helper helped her hamburger help her make a great meal
+	atmos_requirements = list("min_oxy" = 0, "max_oxy" = 0, "min_tox" = 0, "max_tox" = 0, "min_co2" = 0, "max_co2" = 0, "min_n2" = 0, "max_n2" = 0)
+	minbodytemp = 0
+	maxbodytemp = 1500
+	faction = list("agoodmeal")
+	movement_type = FLYING
+	pressure_resistance = 200
+	del_on_death = TRUE

```

Comment:
make him drop a pair of white gloves upon death

Reply:
also set his gender be neutral (or male or whatever)

Reply:
I'm only implementing male hands since the meat proboscis is kinda core to the gameplay rn